mainsail/umbrel-app.yml

```
@@ -0,0 +1,25 @@
+manifestVersion: 1
+id: mainsail
+category: files
+name: Mainsail
+version: "2.13.1"
+tagline: UI for Klipper. Control and monitor your printer from everywhere, from any device.
+description: >-
+  Easy to use. The focus lies on both anticipating what users may need to do and ensuring that the user interface contains elements that are easily accessible, understandable, and user-friendly to make those actions easier.
+
+  Always one step ahead. We work closely with developers of other projects so that functions can already be implemented early on.
+
+  100% open source. Anyone can join, anyone can contribute.
```

Adapt the description a little bit to make it more readable.
```suggestion
tagline: Control and monitor your printer from everywhere, from any device.
description: >-
  🖨️ A modern and responsive user interface for Klipper. Control and monitor your printer from everywhere, from any device.


  **Easy to use.** The focus lies on both anticipating what users may need to do and ensuring that the user interface contains elements that are easily accessible, understandable, and user-friendly to make those actions easier.


  **Always one step ahead.** We work closely with developers of other projects so that functions can already be implemented early on.


  **100% open source.** Anyone can join, anyone can contribute.
```